src/styles/_card.scss

```diff
@@ -0,0 +1,124 @@
+.container {
+  display: grid;
+  justify-content: center;
+
+  $grid-columns: 1;
+
+  grid-template-columns: repeat($grid-columns, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    $grid-columns: 2;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+
+  @media (min-width: 768px) {
+    $grid-columns: 3;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    $grid-columns: 4;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: all 300ms linear;
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $accent-color;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: $hover-color;
+    }
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $label-color;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+      color: $label-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $accent-color;
```

Comment:
Verify that `$accent-color` is defined in `_variables.scss` to ensure that it matches the design specifications and theme.